test/zIntegrationTests.ts

```diff
@@ -40,21 +41,21 @@ function getFloatFromBigNum (bigNumValue) {
 }
 
 async function printPoolState (poolInstance) {
-    console.log(
+    logger(
         "Pool has %s coin0/lvUSD and %s coin1/3CRV",
         getFloatFromBigNum(await poolInstance.balances(0)),
         getFloatFromBigNum(await poolInstance.balances(1)),
     );
 }
 
 async function printPositionState (_r, _positionId, overviewMessage = "Printing Position State") {
-    await console.log(overviewMessage);
+    logger(overviewMessage);
```

Comment:
might need `await` here

Reply:
logger is synchronous so await has no effect